x-pack/metricbeat/module/gcp/gke/_meta/fields.yml

```diff
@@ -0,0 +1,214 @@
+      - name: request_bytes.value
+        type: long
+        description: Memory request of the container in bytes. Sampled every 60 seconds.
+          After sampling, data is not visible for up to 120 seconds.
+      - name: request_utilization.value
+        type: double
+        description: The fraction of the requested memory that is currently in use
+          on the instance. This value can be greater than 1 as usage can exceed the
+          request. Sampled every 60 seconds. After sampling, data is not visible for
+          up to 240 seconds.
+      - name: used_bytes.value
+        type: long
+        description: Memory usage in bytes. Sampled every 60 seconds.
+    - name: ''
+      type: group
+      fields:
+      - name: restart_count.value
+        type: long
+        description: Number of times the container has restarted. Sampled every 60
+          seconds. After sampling, data is not visible for up to 120 seconds.
+      - name: uptime.value
+        type: double
+        description: Time in seconds that the container has been running. Sampled
+          every 60 seconds.
+  - name: node
+    type: group
+    fields:
+    - name: cpu
+      type: group
+      fields:
+      - name: allocatable_cores.value
+        type: double
+        description: Number of allocatable CPU cores on the node. Sampled every 60
+          seconds.
+      - name: allocatable_utilization.value
+        type: double
+        description: The fraction of the allocatable CPU that is currently in use
+          on the instance. Sampled every 60 seconds. After sampling, data is not visible
+          for up to 240 seconds.
+      - name: core_usage_time.value
+        type: double
+        description: Cumulative CPU usage on all cores used on the node in seconds.
+          Sampled every 60 seconds.
+      - name: total_cores.value
+        type: double
+        description: Total number of CPU cores on the node. Sampled every 60 seconds.
+    - name: ephemeral_storage
+      type: group
+      fields:
+      - name: allocatable_bytes.value
+        type: long
+        description: Local ephemeral storage bytes allocatable on the node. Sampled
+          every 60 seconds.
+      - name: inodes_free.value
+        type: long
+        description: Free number of inodes on local ephemeral storage. Sampled every
+          60 seconds.
+      - name: inodes_total.value
+        type: long
+        description: Total number of inodes on local ephemeral storage. Sampled every
+          60 seconds.
+      - name: total_bytes.value
+        type: long
+        description: Total ephemeral storage bytes on the node. Sampled every 60 seconds.
+      - name: used_bytes.value
+        type: long
+        description: Local ephemeral storage bytes used by the node. Sampled every
+          60 seconds.
+    - name: memory
+      type: group
+      fields:
+      - name: allocatable_bytes.value
```

Comment:
Try to follow the Beats naming convention: https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html

Reply:
Should be `max_bytes` or `allocatable.bytes`?

Reply:
In this case - `memory.allocable.bytes`. But this comment applies to all fields you submitted.

Reply:
silly question, it should be `allocatable.bytes` :) Will update all occurrences